cent/client/session/aiohttp.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+from typing import Optional, TYPE_CHECKING, cast, Any
+
+from aiohttp import ClientSession, ClientError
+
+from cent.client.session.base_async import BaseAsyncSession
+from cent.methods.base import CentMethod, CentType
+from cent.exceptions import CentNetworkError
+from cent.methods.batch import BatchMethod
+
+if TYPE_CHECKING:
+    from cent.client.async_client import AsyncClient
+
+
+class AiohttpSession(BaseAsyncSession):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self._session: Optional[ClientSession] = None
+
+    async def _create_session(self) -> ClientSession:
+        if self._session is None or self._session.closed:
+            self._session = ClientSession(headers=self._headers)
+
+        return self._session
+
+    async def close(self) -> None:
+        if self._session is not None and not self._session.closed:
+            await self._session.close()
+
+            # https://docs.aiohttp.org/en/stable/client_advanced.html#graceful-shutdown
+            await asyncio.sleep(0)
+
+    async def make_request(
+        self,
+        client: "AsyncClient",
+        method: CentMethod[CentType],
+        timeout: Optional[float] = None,
+    ) -> CentType:
+        session = await self._create_session()
+        session.headers["X-API-Key"] = client.api_key
+
+        if isinstance(method, BatchMethod):
+            json_data = self.get_batch_json_data(method)
+        else:
+            json_data = method.model_dump(exclude_none=True)
+
+        url = f"{self._base_url}/{method.__api_method__}"
+
+        try:
+            async with session.post(
+                url=url,
+                json=json_data,
+                timeout=timeout or self._timeout,
+            ) as resp:
+                raw_result = await resp.text()
+        except asyncio.TimeoutError:
+            raise CentNetworkError(method=method, message="Request timeout") from None
```

Comment:
I think it's important to distinguish between timeouts and network errors. Timeout may originate from network, or from Centrifugo itself. Maybe have separate?